### PR TITLE
new qoutes added

### PR DESCRIPTION
**Pull request by binismail**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by the author's first name and not ❌ duplicates.
- [x] The sentence attribute does not have inner quotation marks and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Hey! Please add quotes related to "learning a foreign language" as suggested in the [contributing guidelines](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

**Comment by binismail** [comment]
okay will do so.

**Comment by binismail** [comment]
changes made @Kvaibhav01

File: server/data/quotes.json [hunk 1]
```diff
@@ -120,8 +125,8 @@
     "sentence": "Learning is a treasure that will follow its owner everywhere."
   },
   {
-    "author": "Chinese Proverb",
-    "sentence": "To learn a language is to have one more window from which to look at the world."
+    "author": "Robert M. Harrison",
+    "sentence": "Learning a foriegn language will not only help you, perfecting your world it will also help you perfect others."
```
Comment (jayehernandez) [inline server/data/quotes.json:129]: Please fix typo on the word **foreign**.

**Review by jayehernandez — CHANGES_REQUESTED** [review]

**Comment by binismail** [comment]
oka

File: server/data/quotes.json (hunk 1 above)
Comment (binismail) [reply server/data/quotes.json]: Done

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @binismail!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Comment by binismail** [comment]
Thank you, @jayehernandez looking forward to contributing more to this amazing project

**Pull request merged**